### PR TITLE
Include a new version of the variant annotation report page

### PR DESCRIPTION
**Pull request by zhx828**
Please ignore few images added. I don't find a good way to include these images from npm package yet.

AwesomeScreenshot-localhost--2019-08-16_4_05

### CODE REVIEW

File: app/scripts/directives/annotationReport.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            scope: {
+                hugoSymbol: '=',
+                alteration: '=',
+                tumorType: '=',
+                data: '=',
+            },
+            link: function postLink(scope) {
+                function processLevel(levelOfEvidence) {
+                    var level = levelOfEvidence.replace('LEVEL_', '');
+                    return '<span class="oncokb level-' + level + '">Level ' + _.upperFirst(level) + '</span>';
```
Comment (victoria34) [inline app/scripts/directives/annotationReport.js]: I didn't see any css called `level-*` in main.scss

Comment (zhx828) [reply app/scripts/directives/annotationReport.js]: @victoria34 it's included in the index.html via cdn

File: app/scripts/directives/annotationReport.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                    scope.summaries = [
+                        scope.data.geneSummary,
+                        scope.data.variantSummary,
+                        scope.data.tumorTypeSummary,
+                        scope.data.diagnosticSummary,
+                        scope.data.prognosticSummary].filter(function(item) {
+                        return !!item;
+                    });
+                });
+                scope.getEvidenceTypeName = function(evidenceType) {
```
Comment (victoria34) [inline app/scripts/directives/annotationReport.js]: This function appear twice in this file and evidencesReport.js. Can you move it in mainUtils.js

File: app/scripts/directives/evidencesReport.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    .directive('evidenceReport', function(mainUtils, _) {
+        return {
+            templateUrl: 'views/evidencesReport.html',
+            restrict: 'E',
+            scope: {
+                shownSections: '=',
+                evidences: '=',
+            },
+            link: function postLink(scope) {
+                scope.getEvidenceTypeName = function(evidenceType) {
```
Comment (victoria34) [inline app/scripts/directives/evidencesReport.js]: Move to mainUtils.js

File: app/scripts/directives/qtip.js [hunk 4]
```diff
@@ -45,8 +48,88 @@ angular.module('oncokbApp')
...
+                                        li.push(subtitle.join(' '));
+                                        li.push('</li>');
+
+                                        content.push(li.join(''));
+                                    });
+                                }
+                                if (_.isArray(scope.abstracts)) {
+                                    _.each(scope.abstracts, function(item) {
+                                        content.push('<li class="list-group-item" style="width: 100%">');
+                                        if (_.isString(item.link)) {
```
Comment (victoria34) [inline app/scripts/directives/qtip.js]: Can we merge those 2 if together? Like 
```
if (_.isString(item.link)) {
      content.push('<a href="' + item.link + '" target="_blank"><b>' + item.abstract + '</b></a>');
} else {
      content.push('<b>' + item.abstract + '</b>');
}
```

**Pull request merged**